spring-integration-file/src/main/java/org/springframework/integration/file/remote/synchronizer/AbstractInboundFileSynchronizer.java

```diff
@@ -227,6 +228,11 @@ public void close() throws IOException {
 
 	@Override
 	public void synchronizeToLocalDirectory(final File localDirectory) {
+		synchronizeToLocalDirectory(localDirectory, Integer.MIN_VALUE);
```

Comment:
Can we introduce already right now `Session.list()` contract with `maxFetchSize`?
Although I guess we can introduce that  afterward on demand 😄